php_pdo_ibm.h

```diff
@@ -72,6 +71,8 @@ ZEND_END_MODULE_GLOBALS(pdo_ibm)
 #define PDO_IBM_G(v) (pdo_ibm_globals.v)
 #endif
 
+#ifdef ZTS
 ZEND_EXTERN_MODULE_GLOBALS(pdo_ibm)
+#endif
```

Comment:
I don't think this is right. This would leave the modules undefined when building under ZTS. Did you mean to put this in `#ifdef PASE` instead?

Reply:
> 
> 
> What's the problem with PHP 8.0.11? AFAICT the biggest issue here the [ulong](https://github.com/php/pecl-database-pdo_ibm/blob/ca2aa20cd6e87e0d794ca09887ef065cc678763e/ibm_driver.c#L1020) which should get replaced with zend_ulong instead of adding a typedef (or just use `unsigned long` instead of `ulong`).
> 
> Then there's some ancillary changes to get rid of the global definitions, which aren't used except on PASE. And then some package.xml version updates, which should be done by the maintainers.

When I was trying to build this package for PHP 8.0.11, found few build issues. Hence given few fixes to resolve the same.